lib/rules/comma-dangle.js

```diff
@@ -128,10 +132,14 @@ module.exports = function(context) {
         var sourceCode = context.getSourceCode();
         var trailingToken = sourceCode.getTokenAfter(lastItem);
         if (trailingToken.value !== ",") {
-            context.report(
-                lastItem,
-                lastItem.loc.end,
-                MISSING_MESSAGE);
+            context.report({
+                node: lastItem,
+                loc: lastItem.loc.end,
+                message: MISSING_MESSAGE,
+                fix: function(fixer) {
+                    return fixer.insertTextAfter(lastItem, ",");
```

Comment:
The problem with that is, that you would always have to run `eslint` a second time right after auto fixing, to ensure no new problems were introduced. Because the first run wouldn’t report those new issues which are created by auto fixing.